BHoM_Engine/Query/Hash.cs

```diff
@@ -37,14 +37,24 @@
 
 namespace BH.Engine.Base
 {
-    public static partial class Compute
+    public static partial class Query
     {
         /***************************************************/
         /**** Public Methods                            ****/
-        /***************************************************/ 
+        /***************************************************/
+
+        [Description("Computes a Hash code for the iObject. The hash uniquely represents an object's state based on its combined properties and their values.")]
+        [Input("iObj", "iObject the hash code should be calculated for.")]
+        public static string Hash(this IObject iObj)
+        {
+            return Hash(iObj, null, null, null, null, 100);
+        }
 
-        [Description("Computes a Hash code for the object which depends solely on its properties.")]
-        [Input("obj", "Objects the hash code should be calculated for.")]
+        /***************************************************/
+
+        [Description("Computes a Hash code for the iObject with exceptions and custom options.\n"
+            + "The hash uniquely identifies the iObject based on the included properties and their values.")]
```

Comment:
```suggestion
            + "The hash uniquely represents an object's state based on the included properties and their values.")]
```

Reply:
Sorry to mirror the other change pushed. Sorry 🙈 

Otherwise looking good now - and neat, to help explain the core principles behind hashes, guids etc. in BHoM going forward